pkg/operator2/starter.go

```diff
@@ -156,6 +157,16 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
+	staleConditions := staleconditions.NewRemoveStaleConditions(
+		[]string{
+			// in 4.1.0 this was accidentally in the list.  This can be removed in 4.3.
+			"Degraded",
```

Comment:
Use the const.

Reply:
> Use the const.

? the const should be deleted.  It's a const that shouldn't exist because any usage of it is wrong.

Reply:
Please follow-up and actually delete the const.